packages/sdk/src/server/index.ts

```
@@ -374,4 +387,34 @@
       method: "GET",
     });
   }
+
+  /**
+   * Makes a proxy request to a URL with the specified query parameters and options.
+   *
+   * @returns A promise resolving to the response from the downstream service
+   */
+  public makeProxyRequest(url: string, query: any, opts: MakeProxyRequestOpts): Promise<any> {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)